includes/DataDump.php

```diff
@@ -11,33 +11,31 @@
 use NullLockManager;
 
 class DataDump {
-	/**
-	 * @return FileBackend
-	 */
-	public static function getBackend() {
+
+	public static function getBackend(): FileBackend {
 		$services = MediaWikiServices::getInstance();
 		$config = $services->getConfigFactory()->makeConfig( 'DataDump' );
 
-		$fileBackend = $config->get( 'DataDumpFileBackend' );
-		if ( $fileBackend != '' ) {
+		$fileBackend = $config->get( ConfigNames::FileBackend );
+		if ( $fileBackend ) {
 			return $services->getFileBackendGroup()->get( $fileBackend );
-		} else {
-			static $backend = null;
-			if ( !$backend ) {
-				$dirConfig = $config->get( 'DataDumpDirectory' );
-				$uploadDir = $config->get( MainConfigNames::UploadDirectory );
-				$backend = new FSFileBackend( [
-					'name'           => 'dumps-backend',
-					'wikiId'         => WikiMap::getCurrentWikiId(),
-					'lockManager'    => new NullLockManager( [] ),
-					'containerPaths' => [ 'dumps-backup' => $dirConfig ?: "{$uploadDir}/dumps" ],
-					'fileMode'       => 0777,
-					'obResetFunc'    => 'wfResetOutputBuffers',
-					'streamMimeFunc' => [ StreamFile::class, 'contentTypeFromPath' ]
-				] );
-			}
+		}
 
-			return $backend;
+		static $backend = null;
+		if ( !$backend ) {
+			$dirConfig = $config->get( ConfigNames::Directory );
+			$uploadDir = $config->get( MainConfigNames::UploadDirectory );
+			$backend = new FSFileBackend( [
+				'name' => 'dumps-backend',
+				'wikiId' => WikiMap::getCurrentWikiId(),
+				'lockManager' => new NullLockManager( [] ),
+				'containerPaths' => [ 'dumps-backup' => $dirConfig ?: "{$uploadDir}/dumps" ],
+				'fileMode' => 0777,
```

Comment:
Please use something more sensible... like 0600 or 0660, please, for the love of-